ports/nethost/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "nethost",
-  "version-date": "2019-12-21",
-  "port-version": 2,
+  "version-date": "2022-05-30",
   "description": "Provides a set of APIs that can be used to host .NET Core (dotnet)",
   "homepage": "https://github.com/dotnet/runtime/tree/master/src/installer/corehost/cli/nethost",
   "supports": "!uwp"
```

```suggestion
  "supports": "!uwp"
  "dependencies": [
    {
      "name": "vcpkg-cmake",
      "host": true
    },
    {
      "name": "vcpkg-cmake-config",
      "host": true
    }
  ]
```
Remember to run the command `.\vcpkg.exe format-manifest  .\ports\nethost\vcpkg.json to format the `vcpkg.json` file after adding these.

---

Thanks for the info!